src/adapters/poi/poi.js

```diff
@@ -32,23 +33,20 @@ export default class Poi {
       this.type === zoomType.type
     )
     if (zoomSetting) {
-      this.zoom = zoomSetting.zoom
+      return zoomSetting.zoom
     } else {
-      this.zoom = DEFAULT_ZOOM
+      return DEFAULT_ZOOM
     }
   }
 
   poiStoreLiteral() {
-    return {
-      latLon: this.latLon,
-      id: this.id,
-      name: this.name,
-      className: this.className,
-      subClassName: this.subClassName,
-      zoom: this.zoom,
-      type: 'poi',
-      bbox: this.bbox,
-    }
+    const serializeKeys = ['id', 'name', 'alternativeName', 'type', 'latLon', 'className', 'subClassName']
```

Comment:
I think we want to store some other details : 
 * `zoom` is currently required to apply the correct zoom level when clicking on a favorite. With this implementation it  goes to zoom 16, instead of 18. It would be worth adding a test case.
 * are we sure that `localName` and other fields are not required ?

Reply:
i agree for zoom, i'm not sure about localName and others fields